lib/smart_answer/calculators/self_assessment_penalties.rb

```diff
@@ -186,12 +193,12 @@ def calculate_interest_for_date(date)
     end
 
     def daily_rate(date)
-      # Rate drops from 3% to 2.75% on 23 August 2016
-      rate_change_date = Date.new(2016, 8, 23)
+      # Rate increased to 3.25% on 21 August 2018
+      rate_change_date = Date.new(2018, 8, 21)
       if date < rate_change_date
         0.03 / 365.0
       else
-        0.0275 / 365.0
+        0.0325 / 365.0
       end
     end
```

Comment:
Not sure about what the policy says, but should we have three options here?
before 23-08-2016 -> 0.03 / 365.0
between 23-08-2016 and 21-08-2018 -> 0.0275 / 365.0
after 21-08-2018 -> 0.0325 / 365.0

Reply:
According to content apparently not, I expect there is also a bit of "is that really worth it" at play here. Considering the interest rate has been wrong since 2018 and no one noticed till now, and it's only an estimate and not used for any real calculations anyway. On the other hand it might just be a HMRC quirk and thats the way it is 🤷‍♂